changelog.d/gh-7787.fixed

```diff
@@ -0,0 +1 @@
+taint analysis: Check function calls without parameters or parenthesis in Ruby
```

Comment:
I guess this a Pro feature (inteproc), maybe add `Pro:` in the changelog to make that clear.

Reply:
Fixed.